.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.DS_Store
```

Comment:
This looks like a change that doesn't necessarily belong to this PR. Could you revert?